test/entrypoint/srv_select.lua

```diff
@@ -192,6 +192,34 @@ package.preload['customers-storage'] = function()
     }
 end
 
+package.preload['statistics-storage'] = function()
+    return {
+        role_name = 'statistics-storage',
+        dependencies = {'cartridge.roles.crud-storage'},
+        init = function()
+            assert(_G._crud.select_on_storage ~= nil)
+
+            -- Here we count requests.
+            local storage_stat_table = {
+                select_requests = 0,
+            }
+
+            -- Wrap select_on_storage() function.
+            local select_on_storage_saved = _G._crud.select_on_storage
+            _G._crud.select_on_storage = function(...)
+                local requests = storage_stat_table.select_requests
+                storage_stat_table.select_requests = requests + 1
```

Comment:
Didn't get it: where is `requests` defined? May be `select_stat_table.select_requests`?